.cargo/config

```diff
@@ -1,5 +1,9 @@
 [build]
 target = "wasm32-unknown-unknown"
 
+[target.'cfg(target_arch = "wasm32")']
+runner = 'wasm-bindgen-test-runner'
```

Comment:
We don't need this as it's not used in CI but moving to `wasm-bindgen-test-runner` (via `cargo install wasm-bindgen-cli`) may not be a bad idea.